src/sentry/static/sentry/app/api.tsx

```diff
@@ -37,6 +37,9 @@ export class Request {
   }
 }
 
+/**
+ * Setup the CSRF + other client early initalization.
```

Comment:
```suggestion
 * Setup the CSRF + other client early initalization
```

just to be consistent :D